dex/elrond_dex_pair/src/liquidity_pool.rs

```diff
@@ -5,6 +5,8 @@ use super::amm;
 use super::config;
 use dex_common::FftTokenAmountPair;
 
+type Nonce = u64;
```

Comment:
Maybe extract type aliases into a "util" crate, if you have such a thing in this project. Makes it so you don't have to keep defining in every file.

Reply:
Done